tests/spread/plugins/craft-parts/kernel/task.yaml

```diff
@@ -0,0 +1,15 @@
+summary: Craft Parts kernel plugin test
+manual: true
+kill-timeout: 180m
+
+systems:
+  - ubuntu-22.04-64
+
+restore: |
+  snapcraft clean --enable-experimental-plugins
+  rm -f ./*.snap
+
+execute: |
+  rm -f ./*.snap
+  snapcraft --enable-experimental-plugins
```

Comment:
I believe this will build in destructive mode, whereas the core20 plugin builds with LXD.  Is that intentional?

Reply:
This is wrong, we want to make sure both can be built using LXD. Fixing.